lib/avrora/codec/object_container_file.ex

```diff
@@ -33,7 +33,7 @@ defmodule Avrora.Codec.ObjectContainerFile do
 
   @impl true
   def decode(payload) when is_binary(payload) do
-    with {:ok, {_, _, decoded}} <- do_decode(payload), do: {:ok, Mapper.to_map(decoded)}
```

Comment:
I'm not sure if this `Mapper.to_map/1` call is still needed so I decided to remove it. If I missed any case in which this function call is useful, I would happy to learn about that :)

Reply:
I think it was used because of misconfiguration. So I guess you can just remove it with specs